# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vlock/disable-pam.patch.data
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- vlock-1.3/Makefile.orig	2003-01-13 22:41:51.000000000 +0100
+++ vlock-1.3/Makefile	2003-01-13 22:42:43.000000000 +0100
@@ -3,8 +3,8 @@
 CC = gcc
 # remove the -DUSE_PAM, -ldl, and -lpam if you aren't using PAM
 RPM_OPT_FLAGS=-O2
-CFLAGS = $(RPM_OPT_FLAGS) -DUSE_PAM
-LDFLAGS = -ldl -lpam -lpam_misc
+CFLAGS = $(RPM_OPT_FLAGS)
+LDFLAGS = -ldl -lcrypt
 
 OBJS = vlock.o signals.o help.o terminal.o input.o
 
